### DISCUSSION TITLE
Vault Complement only from Linked files

### DISCUSSION THREAD

**Discussion by ybd135** [tadashi-aikawa/obsidian-various-complements-plugin#241] [Ideas]
I think a way to make the complement from other files more useful would be to add an option to only use words from files that are linked/backlinked to the current file. If files are linked they probably deal with the same subject, so it's much more likely they will share terms

**Comment by tadashi-aikawa** [comment]
Hi, @ybd135 :)

Please create an issue as described in the README 😉 (I )

> Feature requests / Bugs
> Please create a new [issue](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/issues).
